examples/ticketing-connector/README.me

```diff
@@ -0,0 +1,39 @@
+## Ticketing Example
+
+The `ticketing-connector` example demonstrates how to use Synqly's Ticketing Connector to interact with a ticketing system. This example uses Jira, but could be adapted to use any of the ticketing systems supported by Synqly.
+
+The `ticketing-connector` example performs the following actions (in order):
+
+- Creates an account in your Synqly Organization
+- Creates an `Integration` in this Account (using Jira as the target Ticketing Provider)
+- Retrieves the list of projects
+- Retrieves the list of remote fields
+- Creates a ticket
+- Creates an attachment in the ticket
+- Downloads an attachment
+- Deletes an attachment in the ticket
+- Creates a comment in the ticket
+- Retrieves all comments in the ticket
+- Deletes a comment in the ticket
+- Updates a ticket
+- Cleans up the created tenant
+
+## Setup
+
+You need the following to run this example:
+
+1. An API token for your Synqly Organization.
+2. A Jira account and API token. Follow [this guide to create a token](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/).
+
+All of this information should be placed in a `config.env` file in this directory or set through environment variables.
```

Comment:
Are `config.env` files a go-ism? Otherwise we've been using `.env.local` in other examples, might be nice to stay consistent but definitely not a biggie.